app/controllers/spree/admin/insights_controller.rb

```diff
@@ -103,8 +103,8 @@ def set_default_pagination
           @pagination_hash = { paginate: false }
           unless params[:paginate] == 'false'
             @pagination_hash[:paginate] = true
-            @pagination_hash[:records_per_page] = params[:per_page].try(:to_i) || Spree::Config[:records_per_page]
-            @pagination_hash[:offset] = params[:page].to_i * @pagination_hash[:records_per_page]
+            @pagination_hash[:records_per_page] = params[:per_page].try(:to_i) || 25
+            @pagination_hash[:offset] = (params[:page].to_i || 1) * @pagination_hash[:records_per_page]
```

Comment:
```
nil.to_i       # will be 0
nil.try(:to_i) # stay nil
```
So I think we maybe want to this:
```suggestion
            @pagination_hash[:offset] = (params[:page].try(:to_i) || 1) * @pagination_hash[:records_per_page]
```

But if the **`params[:page]`** is an empty string then it doesn't matter, because of the **`0 || 1`** always **`0`**